locations/Bahraich/inputs/generation/generation_inputs.yaml

```diff
@@ -1,18 +1,3 @@
----
```

Comment:
@paulharfouche , please create a new location from Bahraich rather than using Bahraich as your default..
By using `Bahraich` for all your testing, you've ended up updating this file, which we don't __really__ want to change 😉

Reply:
I did create a new location  a month ago and all my testing is being done on Beirut. I think I changed this file before and never rearranged it sorry! 👍

Reply:
No worries - we'll want to put it back at some point, so we can keep this thread open :smile: